backend/functions/lib/controllers/cases/cases.controller.ts

```diff
@@ -32,7 +32,7 @@ export class CasesController implements CamsController {
       if (context.request.method === 'GET' && context.request.params.caseId) {
         data = await this.getCaseDetails({ caseId: context.request.params.caseId });
       } else {
-        data = await this.searchCases(context.request);
+        data = await this.searchCases(context.request, context.request.query ?? {});
```

Comment:
This seems kind of odd if we are already sending context.request, then why also send context.request.query?

Reply:
We can certainly change this.